### PR TITLE
Beta function

### PR DESCRIPTION
**Pull request by abhinavagarwal07**
https://github.com/symengine/symengine/pull/707

### CODE REVIEW

**Comment by certik** [comment]
+1, this looks good to me. @isuruf any other comments?

File: symengine/eval_mpfr.cpp [hunk 1]
```diff
@@ -208,6 +208,11 @@ class EvalMPFRVisitor : public BaseVisitor<EvalMPFRVisitor> {
         mpfr_gamma(result_, result_, rnd_);
     };
 
+    void bvisit(const Beta &x) {
+        apply(result_, *(x.rewrite_as_gamma()));
+        mpfr_gamma(result_, result_, rnd_);
```
Comment (isuruf) [inline symengine/eval_mpfr.cpp]: This line is not needed

**Comment by isuruf** [comment]
Left one comment. Otherwise +1

**Comment by abhinavagarwal07** [comment]
I think travis needs a rebuild.

**Comment by certik** [comment]
I restarted it. I've seen this error today already. I am not sure what is causing it, if it is our problem or Travis' problem.

**Comment by abhinavagarwal07** [comment]
@certik ping

**Comment by certik** [comment]
Thanks!

**Comment by abhinavagarwal07** [comment]
Thanks

**Pull request merged**